### PR TITLE
Issue 298 markers default size

### PR DESCRIPTION
**Pull request by doutriaux1**
@danlipsa @aashish24 please review. Blines at: https://github.com/UV-CDAT/uvcdat-testdata/pull/184

Thre goal of this PR is to make all markers have similar size for a given value of the marker size, no matter what the marker type is (at the present the hurrican was way bigger)
NOTE: there is an issue with cross and plus markers they do not scale properly. I got them t be all roughly of the same size for a marker size of 1. But when changing to size 5 or 10 all markers scale similarly exceot for plus and cross. The SetScale option on the vtkGlyphSource2D object seems to behave differently for the "Cross" glyph type. At @aashish24 I played with SetScale2 but to no luck so far.

Most striking example can be seen at:
https://github.com/UV-CDAT/uvcdat-testdata/pull/184/files#diff-e2de5072ad476a7a887c81f48602561b
or
https://github.com/UV-CDAT/uvcdat-testdata/pull/184/files#diff-4bc0717af7c272ffff5c6fbc4eacaa35

### CODE REVIEW

**Comment by danlipsa** [comment]
@doutriaux1 @aashish24 Take a look my commit. Seems that the special scaling was caused by code in vcs. I uploaded new baselines that look right.

**Comment by doutriaux1** [comment]
@danlipsa it was here on purpose, otherwise for size one they're ridiculously small. try different size like 20 next, you'll see it does not seem to scale properly.

File: tests/test_vcs_1D_mintics.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ def testMintics(self):
         line.linecolor = "red"  # Red color obviously
         line.linewidth = 2.  # A bit thicker
         line.marker = "circle"  #
-        line.markersize = 10  # A bit bigger
+        line.markersize = 1
         s = MV2.array(s)
```
Comment (danlipsa) [inline tests/test_vcs_1D_mintics.py]: Is markersize 1 the default?

File: tests/test_vcs_wmo_markers.py [hunk 2]
```diff
@@ -20,7 +20,7 @@ def testVCSWMO(self):
         m.worldcoordinate = [0, M, 0, M]
         m.type = wmo
         m.color = [242, ]
-        m.size = [10., ]
+        m.size = [20., ]
         xs = []
```
Comment (danlipsa) [inline tests/test_vcs_wmo_markers.py]: Why isn't this one set to 1 as well and change everything else proportionally?

**Comment by danlipsa** [comment]
test_vcs_markers_uniform_2
@doutriaux1 Here are the images for 5 and 10. They seem to be OK.
test_vcs_markers_uniform_2

**Comment by doutriaux1** [comment]
@danlipsa I don't know why it wasn't working for me! w/o *=10 it was the tiniest markers ever... If you're happy with the PR as is please merge in

**Comment by danlipsa** [comment]
Sounds good.

**Pull request merged**